src/lang-promql/complete/hybrid.ts

```diff
@@ -134,17 +132,33 @@ export class HybridComplete implements CompleteStrategy {
       // So we can autocomplete the labelValue
       return this.autocompleteLabelValue(tree.parent, tree, pos, context, state);
     }
+    if (tree.type.id === LabelMatcher && tree.firstChild?.type.id === LabelName && tree.lastChild !== null) {
```

Comment:
isn't better like that:
```typescript
tree.lastChild?.name === '/!\' 
```

Reply:
I changed it to `tree.lastChild?.type.id === 0` instead, like further down below, so I don't have to compare to some weird error emoji :)

I also added another condition to stop autocompleting multiple match ops after another...